### PR TITLE
Separate Spring Datasources

### PR DESCRIPTION
**Pull request by KochTobi**
Adds different datasources for 

- `life.qbic.finance`
- `life.qbic.identity`
- `life.qbic.projectmanagement`

With `life.qbic.projectmanagement` being the primary datasource

### CODE REVIEW

**Comment by KochTobi** [comment]
tables are created and application works BUT: ProjectOverviewLookup does not find any project anymore. need to investigate here.

**Comment by KochTobi** [comment]
> tables are created and application works BUT: ProjectOverviewLookup does not find any projecst anymore. need to investigate here.

A database view was missing as it turned out.

File: user-interface/src/main/java/life/qbic/datamanager/Application.java [hunk 1]
```diff
@@ -42,8 +41,7 @@
     shortName = "Data Manager",
     offlineResources = {"images/logo.png"})
 @NpmPackage(value = "line-awesome", version = "1.3.0")
-@ComponentScan({"life.qbic"})
-@EntityScan(basePackages = "life.qbic")
```
Comment (sven1103) [inline user-interface/src/main/java/life/qbic/datamanager/Application.java:46]: so this was preventing the data sources from being found?

**Review by sven1103 — APPROVED** [review]
Really good work @KochTobi !

File: user-interface/src/main/java/life/qbic/datamanager/Application.java (hunk 1 above)
Comment (KochTobi) [reply user-interface/src/main/java/life/qbic/datamanager/Application.java:46]: Yes. It created the datasources and injected them prior to the specific configurations taking place.

**Pull request merged**